### DISCUSSION TITLE
Is there a windows version?

### DISCUSSION THREAD

**Discussion by devicemodder** [immich-app/immich#11785] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Would be nice to have a windows Version, as i have a NAS full of pictures that runs windows 10. 

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
There is not, and won't be. Immich requires some dependencies that only run on Linux, and we only support docker as a distribution method (which doesn't run on Windows).

**Comment by mmomjian** [comment]
Just for clarity, you can run Immich on Windows using Docker Desktop or Docker in WSL2.

**Comment by jonesiw396** [comment]
The issue is that not everyone is competent in using Docker Desktop or in troubleshooting the issues in Linux. Nor does everyone want a run a separate workstation just for Linux.

My biggest issue is that after a lot of trial and error, I finally got it to run and see my NAS. Unfortunately, the last Docker update must have broken something. After a reboot, it can't see the files on my NAS. There's got to be a better way to allow it to access mapped drives on Windows.

I get that there are dependencies; however, I'm reminded of the days when Apple refused access to iTunes and see this restriction as a limitation on adoption.

**Reply by devicemodder** [reply]
So... what should I do then, as installing another OS is out of the question and not happening.  Could just  go the route homebridge took and have a bootable image, ready to go to run in virtualbox or hyper-v manager...

**Reply by jonesiw396** [reply]
In my mind, unless you are really familiar with linux and docker desktop, you're going to have a bad time. 

Immich is not for us.

**Comment by devicemodder** [comment]
I am indeed familiar with linux. Locking down immich to a certain OS.
Should not be allowed at all if it's open source software.

**Reply by jonesiw396** [reply]
My comment should really have been a reply to either of the other two. 

Like bo0tzz, they built it on dependencies for linux. It would be nice if Windows installations were a consideration too.
